### ISSUE TITLE
Upload failed

### ISSUE THREAD

**Issue by GlauberF** [xlayers/xlayers#153]
upload sketch file crashing

![deepin screenshot_selecionar area_20190110133553](https://user-images.githubusercontent.com/11314585/50979182-1b4efc80-14dd-11e9-9615-ac854340b3f4.png)

![deepin screenshot_selecionar area_20190110133608](https://user-images.githubusercontent.com/11314585/50979193-1f7b1a00-14dd-11e9-90ba-a2acddb5e5be.png)

**Comment by manekinekko** [comment]
Hi @GlauberF can you share that sketch file with us so we can integrate it into our test suite?

**Comment by manekinekko** [comment]
Ref: here is a similar file to reproduce this bug
> https://www.sketchappsources.com/resource/download-1918.html

**Comment by manekinekko** [comment]
Here is the metadata extracted from sketchtool:

```
{
  "commit" : "5d9b71e4c7d3c5d87bf3c8d3a0faad245becfe2e",
  "length" : 261,
  "fonts" : [
    "HelveticaNeue-Light",
    "Lato-Bold",
    "AvenirNext-Regular",
    "Lato-Black",
    "Lato-Regular",
    "Lato-Semibold",
    "HelveticaNeue"
  ],
  "app" : "com.bohemiancoding.sketch3",
  "appVersion" : "3.6.1",
  "build" : 26313,
  "created" : {
    "app" : "com.bohemiancoding.sketch3",
    "commit" : "5d9b71e4c7d3c5d87bf3c8d3a0faad245becfe2e",
    "version" : 78,
    "appVersion" : "3.6.1",
    "variant" : "NONAPPSTORE",
    "build" : 26313
  },
  "version" : 78,
  "saveHistory" : [
    "NONAPPSTORE.26313"
  ],
  "autosaved" : 1,
  "variant" : "NONAPPSTORE"
}
```

**Comment by GlauberF** [comment]
This is the file I'm trying to

[amazing_ui_kit-commerce_0.1.sketch.zip](https://github.com/xlayers/xlayers/files/2759451/amazing_ui_kit-commerce_0.1.sketch.zip)

**Comment by Shikanime** [comment]
There was a try-catch lower but the JSZip exception does not seem to be handled and is threw globally by ZoneJS according to the stack trace.

**Comment by manekinekko** [comment]
Fixed in #159
